integrations/spiral/src/Bootloader/SearchBootloader.php

```diff
@@ -35,8 +34,21 @@
  */
 final class SearchBootloader extends Bootloader
 {
-    protected const DEPENDENCIES = [
-        ConsoleBootloader::class,
+    protected const SINGLETONS = [
+        AdapterFactory::class => [self::class, 'createAdapterFactory'],
+        EngineRegistry::class => [self::class, 'createEngineRegistry'],
+        Engine::class => [self::class, 'defaultEngine'],
```

Comment:
No this is not the way how it should work and so it should be kept that way, as before that always a default engine exist. As it should be handled in all integrations the same way.